packages/next/next-server/lib/head.tsx

```diff
@@ -66,6 +66,7 @@ function unique() {
   const metaCategories: { [metatype: string]: Set<string> } = {}
 
   return (h: React.ReactElement<any>) => {
+    if (Object.values(h.props).includes(undefined)) return
```

Comment:
This is incorrect as it completely skips the element when one property is undefined, which would be super unexpected. 

Eg 

```
<meta name="something" content={undefined}>
```

Would result in it not being rendered.

Reply:
Could you also add a test to `test/integration` for this behavior.

Reply:
You’re totally right—honestly I just misread that & wrote the code for it 😅 Fixed now.